clang/test/SemaCXX/noexcept-destroying-delete.cpp

```diff
@@ -0,0 +1,21 @@
+// RUN: %clang_cc1 -fsyntax-only -verify -fcxx-exceptions -Wno-unevaluated-expression -std=c++20 %s
```

Comment:
We already have 

https://github.com/llvm/llvm-project/blob/main/clang/test/CXX/expr/expr.unary/expr.unary.noexcept/sema.cpp

maybe the test is better there and wrapped in a `GH118660` namespace?

Reply:
That test file has a lot of dynamic exception specification code in it, so it's leaning heavily on `-std=c++11`.